### PR TITLE
Fix OS X build

### PR DESCRIPTION
**Pull request by eile**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    - PROJECT_NAME=${PWD##*/}
+    - CMAKE_PREFIX_PATH=/usr/local/opt/qt5:$CMAKE_PREFIX_PATH
+  matrix:
+  - BUILD_TYPE=Debug
+  - BUILD_TYPE=Release
+before_install:
+  - brew update
+  - brew outdated cmake || brew upgrade cmake
+  - brew install cppcheck doxygen ninja
+  - brew install zeromq python qt5 homebrew/science/hdf5
```
Comment (tribal-tec) [inline .travis.yml]: no ITK install???

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  - pip install pyparsing
+  - ssh-keyscan bbpcode.epfl.ch >> ~/.ssh/known_hosts
```
Comment (tribal-tec) [inline .travis.yml]: ??

File: .travis.yml (hunk 1 above)
Comment (eile) [inline .travis.yml]: Forgotten. Will fix.

File: .travis.yml (hunk 2 above)
Comment (eile) [inline .travis.yml]: otherwise the lfp functor checkout will block

**Comment by eile** [comment]
Build failure is due to boost::archive trouble. Once this is fixed, this should build.

File: .travis.yml (hunk 2 above)
Comment (tribal-tec) [inline .travis.yml]: that should fail eventually, same as if you clone a non-existing URL

Comment (eile) [inline .travis.yml]: It doesn't. It waits for the user to say y/n to add the key to the known_hosts indefinitely.

**Pull request merged**